### PR TITLE
Adding create composite primary key on sqllite table

### PR DESCRIPTION
**Pull request by poojamat**
*Issue #, if available:*

*Description of changes:*
Adding code to create composite primary key on sqllite table when the Model has Index named "undefined".

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteCommandFactory.java [hunk 1]
```diff
@@ -515,4 +513,26 @@ private StringBuilder parseForeignKeys(SQLiteTable table) {
...
+        final List<String> indexFields = modelSchema.getPrimaryIndexFields();
+        if (indexFields.size() > 0) {
+            builder.append("PRIMARY KEY")
+                    .append(SqlKeyword.DELIMITER)
+                    .append("(");
+            int numIndexFieldsAdded = 0;
+            for (String filed : indexFields) {
+                numIndexFieldsAdded++;
+                builder.append(SqlKeyword.DELIMITER)
+                        .append(filed);
```
Comment (tjleing) [inline aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteCommandFactory.java:529]:
```suggestion
            for (String field : indexFields) {
                numIndexFieldsAdded++;
                builder.append(SqlKeyword.DELIMITER)
                        .append(field);
```

File: aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteCommandFactory.java [hunk 2]
```diff
@@ -515,4 +513,26 @@ private StringBuilder parseForeignKeys(SQLiteTable table) {
...
+                if (numIndexFieldsAdded != indexFields.size()) {
+                    builder.append(",");
+                }
+            }
+            builder .append(")");
```
Comment (tjleing) [inline aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteCommandFactory.java:534]:
```suggestion
            builder.append(")");
```

Comment (tjleing) [reply aws-datastore/src/main/java/com/amplifyframework/datastore/storage/sqlite/SQLiteCommandFactory.java:534]: I assume this was to line up with the `append()`s above the for loop?  Not sure how style works on this team, but this looks better to my eyes 😅

**Review by tjleing — COMMENTED** [review]
Logic looks good, just a few style comments! 😄

**Review by sdhuka — APPROVED** [review]
LGTM!

**Pull request merged**